worker/addfmusite/createFMUTags.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os
+import json
+import uuid
+from pyfmi import load_fmu
+
+#1.0 setup the inputs
+fmupath = sys.argv[1]
+fmu_upload_name = sys.argv[2]
+jsonpath = sys.argv[3]
+fmu = load_fmu(fmupath)
+
+#2.0 get input/output variables from the FMU
+input_names = fmu.get_model_variables(causality = 2).keys()
+output_names = fmu.get_model_variables(causality = 3).keys()
+
+#print ("input names are: )))))): ", input_names )
```

Comment:
I suggest go ahead and remove this comments.

Reply:
resolved.